### PR TITLE
Migrate to TOML format

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Migrate to `toml` format. 

- [x] Latest scanner now detects the license.txt which need to be handled via @susnux's little helper creating actual .license files
- [x] remove license txt files
- [x] fix unprecise license of photos
- [x] add additional licenses comign in via compiled js dependencies

### CODE REVIEW

**Review by susnux — APPROVED** [review]

File: webpack.js [hunk 1]
```diff
@@ -87,4 +88,17 @@ webpackConfig.plugins.push(
...
+		override: {
+			// TODO: Remove if they fixed the license in the package.json
+			'@nextcloud/axios': 'GPL-3.0-or-later',
+			'@nextcloud/vue': 'AGPL-3.0-or-later',
+			'nextcloud-vue-collections': 'AGPL-3.0-or-later',
+		}
+	}),
+]
+
 module.exports = webpackConfig
```
Comment (susnux) [inline webpack.js:104]: still needs to disable the extract comments part of the terser minifier to remove the `.license.txt` files

Comment (AndyScherzinger) [reply webpack.js:104]: 👍
Yes, just need to find the config snippet again 😅

Comment (AndyScherzinger) [reply webpack.js:114]: found it 😃

**Pull request merged**